scripts/setDefaultIndex.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/sh
+
+XHEAD_PARAMS="-i"
+KIBANA_PREFIX="localhost:9200/.kibana"
+NOT_FOUND="not found"
+INDEX_PATTERN="{\"title\": \"network_*\", \"timeFieldName\": \"@TimeUpdated\", \"customFormats\": {}}"
+DEFAULT_INDEX="{\"doc\":{\"doc\":{\"defaultIndex\":\"network_*\"},\"defaultIndex\":\"network_*\"}}"
+JSON_FLAG="-d"
+
+# Create the index pattern and specify TimeUpdated as the default time parameter
+curl -XPOST localhost:9200/.kibana/index-pattern/network_*/?op_type=create -d "$INDEX_PATTERN"
+
+# Check to see if the kibana/config index exists already
+#   If it does not, create it before updating it
+config_exists=$(curl "$XHEAD_PARAMS" -XHEAD "$KIBANA_PREFIX"/config | grep -i "$NOT_FOUND")
+if [ $? == 0  ]; then
+   echo "Config record DOES NOT exist"
+   curl -XPUT "$KIBANA_PREFIX"/config/4.1.4/_create "$JSON_FLAG" "$DEFAULT_INDEX"
```

Comment:
In theory, we should only need the ```_create``` if the config record doesn't exist, since we are passing the parameters as a .json. However, in all my testing, this created the config record, but did not appropriately set the default index pattern as network_*.

I always needed the ```_create``` first, and then the ```_update``` afterwards or the default index pattern did not get set.